providers/discord/src/airflow/providers/discord/hooks/discord_webhook.py

```diff
@@ -119,6 +136,24 @@ def _get_webhook_endpoint(self, http_conn_id: str | None, webhook_endpoint: str
 
         return endpoint
 
+    def _validate_embeds(self, embeds: list[dict[str, Any]]) -> None:
+        """
+        Validate the structure of the embeds.
+
+        :param embeds: List of embed objects to validate
+        :raises AirflowException: If any embed object is invalid
+        """
+        for embed in embeds:
+            if not isinstance(embed, dict):
+                raise AirflowException("Each embed must be a dictionary.")
```

Comment:
Let's create a specific exception instead of using `AirflowException`. e.g., https://github.com/apache/airflow/blob/main/providers/amazon/src/airflow/providers/amazon/aws/exceptions.py